YoutubeExplode/Models/Playlist.cs

```diff
@@ -5,53 +5,53 @@
 namespace YoutubeExplode.Models
 {
     /// <summary>
-    /// Playlist
+    /// Information about a YouTube playlist.
     /// </summary>
     public partial class Playlist
     {
         /// <summary>
-        /// ID
+        /// ID of this playlist.
         /// </summary>
         public string Id { get; }
 
         /// <summary>
-        /// Type
+        /// Type of this playlist.
         /// </summary>
         public PlaylistType Type { get; }
 
         /// <summary>
-        /// Title
+        /// Author of this playlist.
         /// </summary>
-        public string Title { get; }
+        public string Author { get; }
 
         /// <summary>
-        /// Description
+        /// Title of this playlist.
         /// </summary>
-        public string Description { get; }
+        public string Title { get; }
 
         /// <summary>
-        /// Author's display name
+        /// Description of this playlist.
         /// </summary>
-        public string Author { get; }
+        public string Description { get; }
 
         /// <summary>
-        /// Statistics
+        /// Statistics of this playlist.
         /// </summary>
         public Statistics Statistics { get; }
 
         /// <summary>
-        /// Videos in the playlist
+        /// Collection of <see cref="PlaylistVideo"/>s contained in this playlist.
         /// </summary>
         public IReadOnlyList<PlaylistVideo> Videos { get; }
 
         /// <summary />
-        public Playlist(string id, string title, string author, string description, Statistics statistics,
+        public Playlist(string id, string author, string title, string description, Statistics statistics,
```

Comment:
True, that's not very nice. How often do people initialize YoutubeExplode's models on their own though?

Reply:
I don't and probably never will, but there might be some people that use some kind of parsers and want to reuse your models so they wouldn't need to change/add additional mappers.

Reply:
In that case it would be a bigger dependency issue, as the constructors may change more drastically, the way it happened to `Video` or `PlaylistVideo` (which was remove altogether). Relying on external library's models is not a very good design decision.

On the other hand, the only reason I have for changing the order here is consistency with other types, where author goes before title.